### ISSUE TITLE
Update picture element to allow aria-hidden

### ISSUE THREAD

**Issue by scottaohara** [ThePacielloGroup/WAI-ARIA-Usage#41]
While we do not think it's a good idea to allow additional roles / aria-* attributes on the picture element, since it is not supposed to be directly exposed to the a11y-api, there are potential use cases for setting the entire picture, and thus the graphics it represents as hidden.  While this should be done with `alt=""` on the fallback image element, allowing `aria-hidden=true` on the `picture` itself should be allowed as this will also allow authors to hide the element.

Action item:
The `aria-hidden` attribute can be specified on the `picture` element as the only allowed aria attribute.

**Comment by gezlemon** [comment]
Thanks, Scott. I've updated the script to allow the `aria-hidden` attribute on the `picture` element.

**Comment by scottaohara** [comment]
still getting the invalid attribute flag when i test

```html
<picture aria-hidden="true"><img src="#" alt="foo"></picture>
```

**Comment by gezlemon** [comment]
Apologies, I updated the script in GitHub but forgot to upload it for the bookmarklet. Should be okay now.

**Comment by scottaohara** [comment]
confirmed that the update has been implemented.  thank you again @gezlemon
